src/components/main/main.jsx

```diff
@@ -19,6 +20,13 @@ function Main1() {
   const [playerName, setPlayerName] = useState("");
   const [joinModal, setJoinModal] = useState(false);
   const toggleJoin = () => setJoinModal(!joinModal);
+  const [invalidKeyPopup, setInvalidKeyPopupHandler] = useState(false);
```

Comment:
remove this along with the import statement in line number 9 if it is not being used anywhere